service/src/main/java/uk/gov/hmcts/reform/fpl/model/Colleague.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+import uk.gov.hmcts.reform.fpl.enums.ColleagueRole;
+
+import static java.util.Optional.ofNullable;
+
+@Data
+@Jacksonized
+@Builder(toBuilder = true)
+public class Colleague {
+
+    private final ColleagueRole role;
+    private final String title;
+    private final String fullName;
+    private final String email;
+    private final String phone;
+    private final String dx;
```

Comment:
What is "dx"? Could we have a more descriptive name?

Reply:
ported from https://github.com/hmcts/fpl-ccd-configuration/blob/2db3a7f6dfda92f6f7c397d86aa0886019bd376a/service/src/main/java/uk/gov/hmcts/reform/fpl/model/Solicitor.java#L14

for details you can have a look here:
https://www.quora.com/In-the-UK-what-is-a-DX-reference-for-a-law-firm-or-solicitor-What-is-it-used-for